### DISCUSSION TITLE
meshctrl with HTTPS

### DISCUSSION THREAD

**Discussion by 12nick12** [Ylianst/MeshCentral#6123] [Issues]
How do I connect via meshctrl with HTTPS?

**Comment by si458** [comment]
Run `node meshctrl.js` it will output all the arguments
`    console.log("  --url [wss://server]        - Server url, wss://localhost:443 is default.");`

**Comment by 12nick12** [comment]
That's for WSS not secured WSS or HTTPS from what I can see.

**Reply by si458** [reply]
wss is https, just like ws is http, it's websockets over http/https

So if ur server is https://meshcentral.mydomain.com
The url would be wss://meshcentral.mydomain.com
